vue-components/src/components/InputWithExtender.vue

```diff
@@ -109,6 +109,14 @@ export default defineComponent( {
 		border-radius: $wikit-InputExtender-border-radius;
 	}
 
+	&__content-default {
```

Comment:
I'm confused. What is the motivation to not add those styles to the `wikit-InputWithExtender__extension` class?

Reply:
that's what I tried originally, but the styles were ignored 🤔  I'll try again with a fresh look after this while

Reply:
done. not sure what made me create a different class originally 🤷‍♀️